executor/mpp_gather.go

```diff
@@ -256,8 +113,8 @@ func (e *MPPGather) appendMPPDispatchReq(pf *planFragment, tasks []*mppTask, isR
 		}
 		req := &kv.MPPDispatchRequest{
 			Data:      pbData,
-			Task:      mppTask.taskInfo,
-			ID:        mppTask.id,
+			Meta:      mppTask.Meta,
```

Comment:
what is the meaning of `Meta`?

Reply:
Refer to the belowing comment in this PR. Meta is indeed a wrapped "RPCContext". I think its essence is "Meta" or at least "RPC Meta", what do you think ?

Reply:
`taskMeta` may be ok

Reply:
I think it's a little redundant. meta in a MPPDsipatchRequest indicates that it's a task meta instead of something other.